build.gradle

```diff
@@ -411,38 +363,15 @@ tasks.clean.doFirst {
     delete "${projectDir}/jni/build"
 }
 
-task buildNmslib(type:Exec) {
-    dependsOn cmakeJniLib
-    def cmakePath = findExecutable("cmake")
-    logger.lifecycle("Using cmake at: ${cmakePath}")
-
-    if (cmakePath.isEmpty()) {
-        throw new GradleException("CMake not found in PATH. Please install CMake.")
-    }
-
-    commandLine cmakePath,
-            '--build', 'jni/build',
-            '--target', 'opensearchknn_nmslib',
-            '--parallel', "${nproc_count}"
-}
```

Comment:
@jmazanec15 any reason for removing this? I remember @peterzhuamazon added this to ensure that release builds are not failing. Would like to know why we are removing it.

Reply:
Oh I see. Let me fix. I just thought it was duplicate code.

Reply:
np. :)

Reply:
Yes we add this specifically to build nmslib with different gccs compares to faiss lib.

cc: @naveentatikonda 

Thanks.